app/models/__init__.py

```diff
@@ -6,3 +6,4 @@
 from user import *  # noqa
 from location import *  # noqa
 from resource import *  # noqa
+from attribute import *  # noqa
```

Comment:
There appears to still be a pep8 issue. For example, `'from user import *' used; unable to detect undefined names`.